annotations/halkyon-annotations/src/main/resources/halkyon-resource.vm

```diff
@@ -27,6 +28,7 @@ import io.sundr.builder.annotations.BuildableReference;
 #set ($plural = "$kind" + "s")
```

Comment:
The problem with all these is that they might end up desynchronized with what actually happens… i.e. the default plural is taken from Pluralize, not simply by adding an `s` at the end. I guess we can cross that bridge when we get there but maybe it might be a good idea to have a lib that could be used in both the default processing that the client does but also in downstream apps (such as dekorate)? Not sure it's even feasible…